spec/tests/honeybee_hvac_spec.rb

```diff
@@ -49,6 +49,8 @@
     object1 = honeybee_obj_1.to_openstudio(openstudio_model)
     expect(object1).not_to be nil
     expect(object1.nameString).to eq 'Passive House HVAC System'
+    expect((object1.additionalProperties.featureNames)).to include ("DisplayName")
+    #expect((object1.displayName.get)).to eq 'Test Name áéíóúüñ¿¡'
```

Comment:
The commented out tests with non English characters are failing I think there is an issue with JSON to Hash serialization. On printing out the encoding my ruby environment is using UTF-8.

Reply:
@macumber do you know how I can retain non-ascii characters when going from honeybee JSON to OpenStudio?
When translating from OpenStudio to JSON - I was able to use this `(construction_set.displayName.get).force_encoding("UTF-8")`. This doesn't however seem to work the other way round for example `os_constr_set.setDisplayName(@hash[:display_name].force_encoding("UTF-8"))`. 

This is the error I see: 

Failure/Error: expect((object1.displayName.get)).to eq 'Ω▒┤∞äñδ¬à'

       expected: "\uAC74\uC124\uBA85"
            got: "\xCE\xA9\xE2\x96\x92\xE2\x94\xA4\xE2\x88\x9E\xC3\xA4\xC3\xB1\xCE\xB4\xC2\xAC\xC3\xA0"

       (compared using ==)

Reply:
I think the error is in retaining non-ascii characters in a ruby hash, because when I try to set a string with special characters it is set as the display name correctly

Reply:
Hey @tanushree04 did you figure this out?  If not, I would guess that the issue is needing to set the encoding on the string to UTF8 before putting it into the honeybee hash or something like that?